tests/py3-ignores.txt

```diff
@@ -27,8 +25,6 @@ tests/test_engine.py
 tests/test_http_cookies.py
 tests/test_logformatter.py
 tests/test_mail.py
-tests/test_pipeline_files.py
-tests/test_pipeline_images.py
```

Comment:
At EuroPython sprint we learned that it is not enough to get tests passing; one really should read all the code in the module and think about each function/method. Unfortunately this makes both porting and reviewing harder.